examples/fetch-http-cats.ts

```diff
@@ -0,0 +1,42 @@
+import { PromiseWrapper, asyncBufferedTransformer } from "../dist";
```

Comment:
hmm this import seems weird from dist?

Reply:
Oops, probably my editor trying to refactor when I moved this into the example folder.